crates/exex/src/model/payload.rs

```diff
@@ -598,4 +594,499 @@ mod tests {
         // The pointer to the data
         assert_eq!(vm.get_maybe(&Relocatable::from((1, 5))), Some(MaybeRelocatable::from((0, 0))));
     }
+
+    #[test]
+    #[allow(clippy::too_many_lines, clippy::cognitive_complexity)]
+    fn test_gen_arg_block() {
+        // Initialize a new block header
+        let header = Header {
```

Comment:
can't we generate a dummy header with `arbitraty` or other methods instead of writing everything by hand?

Reply:
if you want to use hard coded values, you'd better write the `assert` directly into cairo file, and not in rust. If you want to do the assert in rust, then it doesn't really make sense to make a handwritten block (see also my comment for the asserts)

Reply:
fixed